src/DelimitedDataParser/DelimitedDataReader.cs

```diff
@@ -37,12 +39,14 @@ internal class DelimitedDataReader : DbDataReader
             Encoding encoding,
             char fieldSeparator,
             bool useFirstRowAsColumnHeaders,
+            bool trimColumnHeaders,
```

Comment:
Worth making optional? Default to `true`?

Reply:
I acknowledge that `useFirstRowAsColumnHeaders` seems equally default-able but has no default..

Reply:
I have now realized that this class is typically not constructed publicly so making the parameter default wouldn't really yield a lot of benefit.